packages/eslint-plugin/tests/rules/prefer-standalone-component/cases.ts

```diff
@@ -0,0 +1,57 @@
+import { convertAnnotatedSourceToFailureCase } from '@angular-eslint/utils';
+import type { MessageIds } from '../../../src/rules/prefer-standalone-component';
+
+const messageId: MessageIds = 'preferStandaloneComponent';
+const suggestAddStandalone: MessageIds = 'suggestAddStandalone';
+
+export const valid = [
+  `
+    @Component({
+      standalone: true,
+    })
+    class Test {}
+  `,
+];
+
+export const invalid = [
```

Comment:
Please could you add some richer, more real-world Component metadata examples, just for completeness (it would also make it easier to see how the fixing affects Component metadata which already has multiple lines)

Reply:
I have added several valid & invalid tests cases with `selector`, `template` and other properties to also test multi-line compatibility